score-rce/src/main/java/io/cloudslang/content/remote/services/RunWindowsCommandService.java

```diff
@@ -0,0 +1,54 @@
+package io.cloudslang.content.remote.services;
+
+import io.cloudslang.content.remote.constants.Constants;
+import io.cloudslang.content.remote.constants.Outputs;
+import io.cloudslang.content.remote.utils.RunWindowsCommandInputs;
+import io.cloudslang.content.remote.utils.CommandUtils;
+
+import java.io.*;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Created by pasternd on 07/06/2016.
+ */
+public class RunWindowsCommandService {
+
+    RunWindowsCommandHelper runWindowsCommandHelper = new RunWindowsCommandHelper();
+
+    public String launchCmdWinCommand(RunWindowsCommandInputs runWindowsCommandInputs) throws IOException {
```

Comment:
@dashapaster This method should be made private.... and put it under the public one...